smart-contracts/contracts/cl-vault/src/vault/swap.rs

```diff
@@ -64,8 +75,12 @@ pub fn swap(
     token_in_amount: Uint128,
     token_in_denom: &String,
     token_out_min_amount: Uint128,
+    token_out_denom: &String,
```

Comment:
token_out_denom seems to be unused as at line https://github.com/quasar-finance/quasar/pull/551/files/e7468c08c21e9d76c212ef3f8a30e0ab6465418b#diff-062b1d3087239d92fdc7297e11aeb1f90572c184792a415feb6479617688268bR93 we are redeclaring it

Reply:
lets get to this when we review the base pr